src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1416,9 +1430,16 @@ emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsi
 	else
 		return NULL;
 
-	if (vector_size == 256 || vector_size == 512)
-		return NULL; 
-		
+	if (vector_size == 256 || vector_size == 512) {
```

Comment:
Should `IsSupported` be handled for `Vector64/256/512` since the set of types is well-known and the same as for `Vector128`, that would allow it to be trivially constant true/false as well.